### PR TITLE
(tlg0011)_all_files_but_008

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated all cts_.xml files to current standards.
Changed STOA Schema so I could edit the files.
Updated headers, structured bibls and also added in a modernization statement to the rest of the English files.

### CODE REVIEW

File: data/tlg0011/tlg001/__cts__.xml [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
-      <ti:description xml:lang="eng">Sophocles, creator; Sophocles with an English translation Vol 2; Storr, Francis, 1839-1919, editor, translator; London, New York: William Heinemann Ltd., The Macmillan Company, 1913.</ti:description>
+      <ti:label xml:lang="grc">Τραχίνιαι</ti:label>
+      <ti:description xml:lang="eng">Sophocles, Volume 2. Storr, Francis, editor. London; New York: William Heinemann Ltd.; The Macmillan Company, 1913.</ti:description>
    </ti:edition>
    
    <ti:translation urn="urn:cts:greekLit:tlg0011.tlg001.perseus-eng3" xml:lang="eng" workUrn="urn:cts:greekLit:tlg0011.tlg001">
       <ti:label xml:lang="eng">Trachiniae</ti:label>
-      <ti:description xml:lang="eng">Sophocles, creator; Sophocles the plays and fragments with critical notes, commentary, and translation in English prose, Part 5 The Trachiniae; Jebb, Richard Claverhouse, Sir, 1841-1905, editor, translator; Cambridge: Cambridge University Press, 1892.</ti:description>
+      <ti:description xml:lang="eng">Sophocles the plays and fragments, Part 5: The Trachiniae. Jebb, Richard Claverhouse, Sir, translator. Cambridge: Cambridge University Press, 1892.</ti:description>
```
Comment (lcerrato) [inline data/tlg0011/tlg001/__cts__.xml:12]: I think this label is Latin

File: data/tlg0011/tlg001/tlg0011.tlg001.perseus-eng3.xml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-   <teiHeader>
+   <teiHeader xml:lang="eng">
       <fileDesc>
          <titleStmt>
-            <title type="work" n="Trach.">Trachiniae (English)</title>
-            
-            <author n="Soph.">Sophocles</author>
-            <editor role="editor" n="Jebb">Sir Richard Jebb</editor>
+            <title>Trachiniae</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg001/tlg0011.tlg001.perseus-eng3.xml:7]: Latin

File: data/tlg0011/tlg001/tlg0011.tlg001.perseus-grc2.xml [hunk 3]
```diff
@@ -30,15 +29,22 @@
                 <biblStruct>
                     <monogr>
                         <author>Sophocles</author>
-                        <title>Sophocles. Vol 2: Ajax. Electra. Trachiniae. Philoctetes With an
-                            English translation by F. Storr. The Loeb classical library, 21. </title>
+                        <title>Sophocles</title>
+                        <title type="sub">Ajax. Electra. Trachiniae. Philoctetes</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg001/tlg0011.tlg001.perseus-grc2.xml:33]: xml:lang=mul

File: data/tlg0011/tlg003/tlg0011.tlg003.perseus-grc2.xml [hunk 4]
```diff
@@ -29,16 +28,22 @@
             <biblStruct>
                <monogr>
                   <author>Sophocles</author>
-                  <title>Sophocles. Vol 2: Ajax. Electra. Trachiniae. Philoctetes With an
-English translation by F. Storr. The Loeb classical library, 21.
-</title>
+                  <title>Sophocles</title>
+                  <title type="sub">Ajax. Electra. Trachiniae. Philoctetes</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg003/tlg0011.tlg003.perseus-grc2.xml:32]: multiple languages here

File: data/tlg0011/tlg004/__cts__.xml [hunk 5]
```diff
@@ -4,13 +4,13 @@
...
    
    <ti:edition xml:lang="grc" workUrn="urn:cts:greekLit:tlg0011.tlg004" urn="urn:cts:greekLit:tlg0011.tlg004.perseus-grc2">
-      <ti:label xml:lang="eng">Oedipus Tyrannus</ti:label>
-      <ti:description xml:lang="eng">Sophocles, creator; Sophocles with an English translation Vol 1; Storr, Francis, 1839-1919, editor, translator. London, New York: William Heinemann Ltd., The Macmillan Company, 1912.</ti:description>
+      <ti:label xml:lang="grc">Οἰδίπους Τύραννος</ti:label>
+      <ti:description xml:lang="eng">Sophocles, Volume 1. Storr, Francis, editor. London; New York: William Heinemann Ltd.; The Macmillan Company, 1912.</ti:description>
    </ti:edition>
    
    <ti:translation urn="urn:cts:greekLit:tlg0011.tlg004.perseus-eng2" xml:lang="eng" workUrn="urn:cts:greekLit:tlg0011.tlg004">
       <ti:label xml:lang="eng">Oedipus Tyrannus</ti:label>
```
Comment (lcerrato) [inline data/tlg0011/tlg004/__cts__.xml:12]: I think this is a Latin title

File: data/tlg0011/tlg004/__cts__.xml [hunk 6]
```diff
@@ -4,13 +4,13 @@
...
-      <ti:description xml:lang="eng">Sophocles, creator; Sophocles the plays and fragments with critical notes, commentary, and translation in English prose Part 1 The Oedipus Tyrannus; Jebb, Richard Claverhouse, Sir, 1841-1905, editor, translator. Cambridge: Cambridge University Press, 1887.</ti:description>
+      <ti:description xml:lang="eng">Sophocles the plays and fragments, Part 1: The Oedipus Tyrannus. Jebb, Richard Claverhouse, Sir, translator. Cambridge: Cambridge University Press, 1887.</ti:description>
```
Comment (lcerrato) [inline data/tlg0011/tlg004/__cts__.xml:13]: multiple languages

File: data/tlg0011/tlg004/tlg0011.tlg004.perseus-eng2.xml [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-   <teiHeader>
+   <teiHeader xml:lang="eng">
       <fileDesc>
          <titleStmt>
-            <title type="work" n="OT">Oedipus Tyrannus (English)</title>
-            
-            <author n="Soph.">Sophocles</author>
-            <editor role="editor" n="Jebb">Sir Richard Jebb</editor>
+            <title>Oedipus Tyrannus</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg004/tlg0011.tlg004.perseus-eng2.xml:7]: Latin title?

File: data/tlg0011/tlg004/tlg0011.tlg004.perseus-eng2.xml [hunk 8]
```diff
@@ -29,14 +29,17 @@
             <biblStruct>
                <monogr>
                   <author>Sophocles</author>
-                  <title>The Oedipus Tyrannus of Sophocles. Edited with introduction and notes by Sir Richard Jebb.</title>
-                  <editor>Sir Richard Jebb</editor>
+                  <title>Sophocles The Plays and Fragments</title>
+                  <title type="sub">The Oedipus Tyrannus</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg004/tlg0011.tlg004.perseus-eng2.xml:33]: Latin?

File: data/tlg0011/tlg004/tlg0011.tlg004.perseus-eng2.xml [hunk 9]
```diff
@@ -29,14 +29,17 @@
...
+                  <editor role="translator">Richard Jebb</editor>
                   <imprint>
                      <pubPlace>Cambridge</pubPlace>
                      <publisher>Cambridge University Press</publisher>
                      <date>1887</date>
                   </imprint>
+                  <biblScope unit="volume">1</biblScope>
                </monogr>
+               <ref target="https://hdl.handle.net/2027/umn.31951002351065d?urlappend=%3Bseq=107%3Bownerid=28332883-106">HathiTrust</ref>
```
Comment (lcerrato) [inline data/tlg0011/tlg004/tlg0011.tlg004.perseus-eng2.xml:42]: also here https://archive.org/details/oedipustyrannus00soph/page/4/mode/2up

File: data/tlg0011/tlg007/tlg0011.tlg007.perseus-grc2.xml [hunk 10]
```diff
@@ -27,16 +26,22 @@
...
-                     <publisher>William Heinemann Ltd.; The Macmillan Company</publisher>
+                     <pubPlace>London</pubPlace>
+                     <pubPlace>New York</pubPlace>
+                     <publisher>William Heinemann Ltd.</publisher>
+                     <publisher>The Macmillan Company</publisher>
                      <date>1912</date>
                   </imprint>
+                  <biblScope unit="volume">1</biblScope>
                </monogr>
+               <series xml:lang="lat">
```
Comment (lcerrato) [inline data/tlg0011/tlg007/tlg0011.tlg007.perseus-grc2.xml:41]: Latin language not needed

File: data/tlg0011/tlg007/tlg0011.tlg007.perseus-grc2.xml [hunk 11]
```diff
@@ -27,16 +26,22 @@
             <biblStruct>
                <monogr>
                   <author>Sophocles</author>
-                  <title>Sophocles. Vol 1: Oedipus the king. Oedipus at Colonus. Antigone.
-With an English translation by F. Storr. The Loeb classical library,
-20. </title>
+                  <title>Sophocles</title>
+                  <title type="sub">Ajax. Electra. Trachiniae. Philoctetes</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg007/tlg0011.tlg007.perseus-grc2.xml:30]: multiple languages

File: data/tlg0011/tlg007/tlg0011.tlg007.perseus-eng2.xml [hunk 12]
```diff
@@ -29,14 +29,18 @@
...
+                  <title type="sub">The Oedipus at Colonus</title>
+                  <editor role="translator">Richard Jebb</editor>
                   <imprint>
                      <pubPlace>Cambridge</pubPlace>
                      <publisher>Cambridge University Press</publisher>
                      <date>1889</date>
                   </imprint>
+                  <biblScope unit="volume">2</biblScope>
                </monogr>
+               <ref target="https://hdl.handle.net/2027/umn.319510023510687?urlappend=%3Bseq=73%3Bownerid=28341244-72">HathiTrust</ref>
```
Comment (lcerrato) [inline data/tlg0011/tlg007/tlg0011.tlg007.perseus-eng2.xml:42]: also here https://archive.org/details/sophocles02soph_0/page/10/mode/2up

File: data/tlg0011/tlg006/tlg0011.tlg006.perseus-grc2.xml [hunk 13]
```diff
@@ -27,16 +26,22 @@
             <biblStruct>
                <monogr>
                   <author>Sophocles</author>
-                  <title>Sophocles. Vol 2: Ajax. Electra. Trachiniae. Philoctetes With an
-English translation by F. Storr. The Loeb classical library, 21.
-</title>
+                  <title>Sophocles</title>
+                  <title type="sub">Ajax. Electra. Trachiniae. Philoctetes</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg006/tlg0011.tlg006.perseus-grc2.xml:30]: multiple languages

File: data/tlg0011/tlg006/tlg0011.tlg006.perseus-grc2.xml [hunk 14]
```diff
@@ -27,16 +26,22 @@
...
-                     <publisher>William Heinemann Ltd.; The Macmillan Company</publisher>
+                     <pubPlace>London</pubPlace>
+                     <pubPlace>New York</pubPlace>
+                     <publisher>William Heinemann Ltd.</publisher>
+                     <publisher>The Macmillan Company</publisher>
                      <date>1913</date>
                   </imprint>
+                  <biblScope unit="volume">2</biblScope>
                </monogr>
+               <series xml:lang="lat">
```
Comment (lcerrato) [inline data/tlg0011/tlg006/tlg0011.tlg006.perseus-grc2.xml:41]: Latin language here?

File: data/tlg0011/tlg006/tlg0011.tlg006.perseus-eng2.xml [hunk 15]
```diff
@@ -29,14 +29,17 @@
...
+                  <title type="sub">The Philoctetes</title>
+                  <editor role="translator">Richard Jebb</editor>
                   <imprint>
                      <pubPlace>Cambridge</pubPlace>
                      <publisher>Cambridge University Press</publisher>
                      <date>1898</date>
                   </imprint>
+                  <biblScope unit="volume">4</biblScope>
                </monogr>
+               <ref target="https://hdl.handle.net/2027/umn.31951002351070k?urlappend=%3Bseq=96%3Bownerid=28348041-95">HathiTrust</ref>
```
Comment (lcerrato) [inline data/tlg0011/tlg006/tlg0011.tlg006.perseus-eng2.xml:42]: https://hdl.handle.net/2027/umn.319510023510687?urlappend=%3Bseq=72%3Bownerid=28341244-71

wrong book here?

File: data/tlg0011/tlg001/__cts__.xml (hunk 1 above)
Comment (lcerrato) [reply data/tlg0011/tlg001/__cts__.xml:12]: the description is multiple languages

File: data/tlg0011/tlg002/tlg0011.tlg002.perseus-eng2.xml [hunk 16]
```diff
@@ -27,16 +27,19 @@
...
+                        <title type="sub">The Antigone</title>
+                        <editor role="translator">Richard Jebb</editor>
                         <imprint>
                             <pubPlace>Cambridge</pubPlace>
                             <publisher>Cambridge University Press</publisher>
                             <date>1891</date>
                         </imprint>
+                        <biblScope unit="volume">3</biblScope>
                     </monogr>
+                    <ref target="https://hdl.handle.net/2027/nyp.33433081618823?urlappend=%3Bseq=92%3Bownerid=27021597765125578-124">HathiTrust</ref>
```
Comment (lcerrato) [inline data/tlg0011/tlg002/tlg0011.tlg002.perseus-eng2.xml:40]: also here https://archive.org/details/sophocles03soph/page/8/mode/2up

(pen marks throughout though)

File: data/tlg0011/tlg005/tlg0011.tlg005.perseus-eng2.xml [hunk 17]
```diff
@@ -29,14 +29,17 @@
...
+                  <title type="sub">The Electra</title>
+                  <editor role="translator">Richard Jebb</editor>
                   <imprint>
                      <pubPlace>Cambridge</pubPlace>
                      <publisher>Cambridge University Press</publisher>
                      <date>1894</date>
                   </imprint>
+                  <biblScope unit="volume">6</biblScope>
                </monogr>
+               <ref target="https://hdl.handle.net/2027/umn.31951002351070k?urlappend=%3Bseq=96%3Bownerid=28348041-95">HathiTrust</ref>
```
Comment (lcerrato) [inline data/tlg0011/tlg005/tlg0011.tlg005.perseus-eng2.xml:42]: also https://archive.org/details/electraeditedbyr00sophuoft/page/2/mode/2up

File: data/tlg0011/tlg006/tlg0011.tlg006.perseus-eng2.xml (hunk 15 above)
Comment (lcerrato) [reply data/tlg0011/tlg006/tlg0011.tlg006.perseus-eng2.xml:42]: also here https://archive.org/details/sophocles04soph/page/6/mode/2up

File: data/tlg0011/tlg008/tlg0011.tlg008.perseus-eng2.xml [hunk 18]
```diff
@@ -5,7 +5,7 @@
       <fileDesc>
          <titleStmt>
             <title>Tracking Satyrs</title>
```
Comment (lcerrato) [inline data/tlg0011/tlg008/tlg0011.tlg008.perseus-eng2.xml:7]: extent not needed
Bill Merrill and Elli Mylonas not needed

File: data/tlg0011/tlg003/tlg0011.tlg003.perseus-grc2.xml [hunk 19]
```diff
@@ -29,16 +28,22 @@
...
-                     <publisher>William Heinemann Ltd.; The Macmillan Company</publisher>
+                     <pubPlace>London</pubPlace>
+                     <pubPlace>New York</pubPlace>
+                     <publisher>William Heinemann Ltd.</publisher>
+                     <publisher>The Macmillan Company</publisher>
                      <date>1913</date>
                   </imprint>
+                  <biblScope unit="volume">2</biblScope>
                </monogr>
+               <series xml:lang="lat">
```
Comment (lcerrato) [inline data/tlg0011/tlg003/tlg0011.tlg003.perseus-grc2.xml:43]: This series got tagged as Latin throughout?

File: data/tlg0011/tlg002/tlg0011.tlg002.perseus-grc2.xml [hunk 20]
```diff
@@ -29,15 +28,22 @@
...
-                            <publisher>William Heinemann Ltd.; The Macmillan Company</publisher>
+                            <pubPlace>London</pubPlace>
+                            <pubPlace>New York</pubPlace>
+                            <publisher>William Heinemann Ltd.</publisher>
+                            <publisher>The Macmillan Company</publisher>
                             <date>1912</date>
                         </imprint>
+                        <biblScope unit="volume">1</biblScope>
                     </monogr>
+                    <series xml:lang="lat">
```
Comment (lcerrato) [inline data/tlg0011/tlg002/tlg0011.tlg002.perseus-grc2.xml:43]: This got tagged as Latin throughout (may not have flagged all)

**Review by lcerrato — CHANGES_REQUESTED** [review]
Lots of language attribute issues and some additional header issues such as Internet Archive versions of texts.

File: data/tlg0011/tlg003/tlg0011.tlg003.perseus-grc2.xml (hunk 19 above)
Comment (AlisonBabeu) [reply data/tlg0011/tlg003/tlg0011.tlg003.perseus-grc2.xml:43]: Duh, how did I miss that?

File: data/tlg0011/tlg002/tlg0011.tlg002.perseus-grc2.xml (hunk 20 above)
Comment (lcerrato) [reply data/tlg0011/tlg002/tlg0011.tlg002.perseus-grc2.xml:43]: The language stuff is so easy to miss in a larger PR. There are so many places to check. I'm sure I missed something.

File: data/tlg0011/tlg008/__cts__.xml [hunk 21]
```diff
@@ -1,15 +1,14 @@
...
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0011.tlg008" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0011">
-   <ti:title xml:lang="lat">Ichneutae</ti:title>
-   <ti:title xml:lang="eng">Tracking Satyrs</ti:title>
-   
+   <ti:title xml:lang="lat">Ichneutae</ti:title>   
    <ti:edition xml:lang="grc" workUrn="urn:cts:greekLit:tlg0011.tlg008" urn="urn:cts:greekLit:tlg0011.tlg008.perseus-grc2">
-      <ti:label xml:lang="eng">Ichneutae, The Oxyrhynchus papyri Part 9 1174</ti:label>
-      <ti:description xml:lang="eng">Sophocles, creator; Grenfell, Bernard P. (Bernard Pyne), 1869-1926, creator; Hunt, Arthur S. (Arthur Surridge), 1871-1934, creator; Grenfell, Bernard P. (Bernard Pyne), 1869-1926, editor; Hunt, Arthur S. (Arthur Surridge), 1871-1934, editor</ti:description>
+      <ti:label xml:lang="lat">Ichneutae</ti:label>
+      <ti:description xml:lang="eng">Sophocles. Oxyrhynchus Papyri, Part 9. Grenfell, Bernard P. (Bernard Pyne); Hunt, Arthur S. (Arthur Surridge), editors.
```
Comment (lcerrato) [inline data/tlg0011/tlg008/__cts__.xml:6]: Do we do this name expansion elsewhere? I don't think so. It's fine if you'd like but seems non-standard.

File: data/tlg0011/tlg008/__cts__.xml [hunk 22]
```diff
@@ -1,15 +1,14 @@
...
+      London: Egypt Exploration Fund, 1912.</ti:description>
    </ti:edition>
    <ti:translation urn="urn:cts:greekLit:tlg0011.tlg008.perseus-eng2" xml:lang="eng" workUrn="urn:cts:greekLit:tlg0011.tlg008">
-      <ti:label xml:lang="eng">Ichneutae, Tracking Satyrs</ti:label>
-      <ti:description xml:lang="eng">Sophocles, creator; Mahoney, Anne (Anne Elizabeth), translator.</ti:description>
+      <ti:label xml:lang="eng">Tracking Satyrs</ti:label>
+      <ti:description xml:lang="eng">Sophocles. Tracking Satyrs. Mahoney, Anne (Anne Elizabeth), translator.</ti:description>
```
Comment (lcerrato) [inline data/tlg0011/tlg008/__cts__.xml:11]: same with the name

**Review by lcerrato — APPROVED** [review]
The only thing I see is the expanded names in the parentheses, so that's your decision to fix or leave as is. Probably nothing in parentheses is needed for cts purposes?

**Pull request merged**